google/ads/google_ads/client.py

```diff
@@ -18,15 +18,14 @@
 import os
 import yaml
 import json
+import grpc
 from collections import namedtuple
 from importlib import import_module
 
-import google.api_core.grpc_helpers
-import google.auth.transport.requests
-import google.oauth2.credentials
-import google.ads.google_ads.errors
-from google.protobuf.json_format import MessageToJson
-import grpc
+from google.auth.transport.requests import Request
```

Comment:
module_name and package_names should be lowercase https://google.github.io/styleguide/pyguide.html#316-naming (I should've caught MessageToJson previously)

Reply:
Unfortunately this is an external module that we can't control, so would you prefer we use `as`, like:

```python
from google.auth.transport.requests import Request as auth_request
```

Reply:
My opinion is that it is probably more confusing to modify the name of modules/classes/etc in an imported library. If they break the style, I don't think it should be up to us to fix it.